src/Resources/contao/library/Contao/Combiner.php

```diff
@@ -212,11 +219,18 @@ public function getFileUrls()
 			{
 				$strPath = 'assets/' . $strTarget . '/' . str_replace('/', '_', $arrFile['name']) . $this->strMode;
 
+				// Load the existing file
+				if (file_exists(TL_ROOT . '/' . $strPath))
```

Comment:
This should be

```php
if (!file_exists(TL_ROOT . '/' . $strPath))
{
    $objFile = new \File($strPath);
    $objFile->write($this->handleScssLess($content, $arrFile));
    $objFile->close();
}
```

But the PR will conflict with the `hotfix/4.4.3` changes anyway once they are merged.